### DISCUSSION TITLE
Sei governor limits

### DISCUSSION THREAD

**Discussion by claudijd** [wormhole-foundation/wormhole#3295] [Governance]
I would like to open up a discussion for Governor parameters for Sei. We have seen significant inbound/outbound flows from Sei since it's launch and a rapidly approaching the governor limit of $500K/$50K. The current value will likely soon limiting outbound transactions from Sei to other chains.

Inflows to Sei since launch are: $2.9M
Outflows from Sei since launch are: $396K
Current governor limit consumption: 94.38% of the $500K daily limit (updated 11:47 EST on August 16th)

I'd like to propose doubling the existing limit from $500K/$50K to $1M/$100K offer some headroom to allow for organic growth.  We will continue to monitor the activity during this discussion and will post back if these expectations change.

I have also uploaded a PR to reflect this proposal to show the actual config changes proposed: https://github.com/wormhole-foundation/wormhole/pull/3296

Update: August 17th @ 7:37AM EST - A number of chains have seen exhausted limits relating to Sei in/out flow, which is changing this guidance...

Sei: 100% utilization (causing delays)
Arbitrum: 99.98% utilization (causing delays)
Optimism: 99.96% utilization (causing delays)

I'm suggesting a 2x of additional flow for Arbitrum and Optimism limits from $1M/$100K and $500K/$50K to $2M/$200K and $1M/$100K respectively.  I will be updating the existing PR for Sei to reflect this config recommendations.

**Comment by philipsu522** [comment]
Hey folks, the foundation does expect significantly more traffic (5-10x) in the next 12 hours which means the chain would likely hit the new limit (2x) quite quickly given that it hit the current limit in <12h. It would would be a pretty bad user experience if they ran into issues getting VAAs again which should be avoided as much as possible. Any way the limit can be bumped by more i.e. 5x?

**Comment by 1xCL** [comment]
With the on-going Sei airdrops using Wormhole, I think we should increase the limit to more than double. I suggest $2M/$200K or $5M/$500K.
